runner_manager/models/runner_group.py

```diff
@@ -268,6 +268,24 @@ def healthcheck(
             self.delete_runner(runner)
             log.info(f"Runner {runner.name} deleted")
 
+    def reset(self, github: GitHub):
+        """Reset runner group."""
+        for runner in self.get_runners():
+            if runner.id is not None:
+                (
+                    github.rest.actions.get_self_hosted_runner_for_org(
+                        self.organization, runner.id
+                    ).parsed_data
+                )
+                runner.update_from_github(github)
+            if not runner.is_active:
```

Comment:
So if the runner is active we do nothing?

Reply:
if the runner is active it is not a good choice to reset it